metrics/default_metrics/tarantool/memory.lua

```diff
@@ -0,0 +1,18 @@
+local utils = require('metrics.default_metrics.tarantool.utils');
+
+local function get_memory_metrics()
```

Comment:
Maybe `update_memory_metrics` instead of `get`?

There is confusion with current naming as it seems the function returns memory_metrics. But it just updates / creates some gauges.